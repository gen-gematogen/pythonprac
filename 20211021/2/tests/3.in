:x x x
x -1
:splitter s s.split('_')
splitter "Hello_world"
:big_FUNCTION x y z m x**y+z**m
big_FUNCTION 1 2 3 0.1
quit 'Have_a_good_day_mate:=),{}_()_{}'