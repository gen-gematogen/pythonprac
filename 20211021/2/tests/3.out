-1
['Hello', 'world']
2.1161231740339046
Have_a_good_day_mate:=),4_()_7
